influx-reports/influx.py

```diff
@@ -0,0 +1,140 @@
+#!/usr/bin/env python3
+import argparse
+import datetime
+import dateutil.parser
```

Comment:
I don't think this is standard lib, so should be added to requirements and constraints